bin/GameIndex.yaml

```diff
@@ -35090,8 +35192,11 @@ SLUS-20758:
   patches:
     03F9C6D1:
       content: |-
-        comment=IPU freeze fix
-        patch=0,EE,00109d04,word,00000000
```

Comment:
Old patches for Growlanser (both discs) probably should stay, my patch is not for IPU issues.